cumulus/polkadot-omni-node/lib/Cargo.toml

```diff
@@ -24,6 +24,7 @@ futures = { workspace = true }
 log = { workspace = true, default-features = true }
 serde = { features = ["derive"], workspace = true, default-features = true }
 serde_json = { workspace = true, default-features = true }
+chain-spec-builder = { workspace = true , package = "staging-chain-spec-builder" }
```

Comment:
Do we need to mention the package? Thought `chain-spec-builder = { workspace = true }` should do it.